### ISSUE TITLE
Add Contact adjustment 

### ISSUE THREAD

**Issue by Franatix93** [Lyall/MGSHDFix#58]
If you add 50% contrast in Photoshop, you can match the Master Collection to the PS3 HD release. Would it be possible to make a contrast adjustment in the .ini file?

![20231112104221_1](https://github.com/Lyall/MGSHDFix/assets/150065841/385cf33c-59fb-4e62-93f5-da1a23386beb)

**Comment by mooms06** [comment]
So, is it possible ?

**Comment by Lyall** [comment]
> So, is it possible ?

Yes it's likely possible. It's just a matter of finding time and motivation to look into it. I can't guarantee when that will be unfortunately. 

What problem are you having with the ReShade solution? That would be the way to go until I or someone else can look in to fixing it through game code.

**Comment by mooms06** [comment]
The mod don't exist for MGS2. For MGS3, I couldn't get it to work, Reshade crashes on launch or won't launch.

**Comment by ShizCalev** [comment]
just making some notes for myself;
https://learn.microsoft.com/en-us/windows/win32/direct3ddxgi/using-gamma-correction

**Comment by ShizCalev** [comment]
Fixing broken issues, ignore this message.

**Comment by ShizCalev** [comment]
Reshade's now fully compatible as of May. Will still look into adding native support for this.
